src/lib/logger.ts

```diff
@@ -202,6 +203,7 @@ export class Logger {
         documentKey: document.key,
         documentType: document.type,
         documentChanges: document.changeCount,
+        section,
```

Comment:
I was worried earlier that this might get logged as `section: "undefined"` or `section: null` in the case where the section doesn't exist, but since we `JSON.stringify()` it before sending it out any `undefined` values should be removed at that point.